mllib/src/main/scala/org/apache/spark/ml/r/GaussianMixtureWrapper.scala

```diff
@@ -124,7 +129,8 @@ private[r] object GaussianMixtureWrapper extends MLReadable[GaussianMixtureWrapp
       val rMetadataStr = sc.textFile(rMetadataPath, 1).first()
       val rMetadata = parse(rMetadataStr)
       val dim = (rMetadata \ "dim").extract[Int]
-      new GaussianMixtureWrapper(pipeline, dim, isLoaded = true)
+      val logLikelihood = (rMetadata \ "logLikelihood").extract[Double]
+      new GaussianMixtureWrapper(pipeline, dim, logLikelihood, isLoaded = true)
```

Comment:
Yeah, I'll link this to SPARK-18864 and collect others' thoughts. If we really need make R model persistence compatibility with old versions, I can address it in follow-up work. Thanks.